build-managed.cmd

```diff
@@ -0,0 +1,2 @@
+@call %~dp0run.cmd build-managed -nodoReuse -binclashWindows %*
```

Comment:
build-native and build-managed are a detail of implementation in this repo. 
For the common dev workflow `Build` is the task that encapsulates this work and that is why Build is calling them.

If all of our .NET Core repos need this behavior, then we can update the dev workflow, but I don't think this is true.